site/content/docs/5.2/getting-started/introduction.md

```diff
@@ -10,6 +10,25 @@ aliases:
   - "/docs/getting-started/introduction/"
 toc: true
 ---
+## About Orange Brand
+
+As Boosted is based on/inherited from Bootstrap, you'll find in Boosted **all** the Bootstrap's components and their variants, but with an Orange branded look.
+
+Among those inherited components and variants, some exist in the Orange Design System and fully respect its design specifications and some don't.Those later ones should not be used in your projects to ensure a consistent user experience throughout our Orange web services.
+
+As for technical reasons we have to keep them in/we can't remove them from this documentation, we tagged them with informational design callouts to inform you of what can be used or not from an Orange design point of view.
+Thanks for taking their information into account.
+
+These callouts look like that (e.g.: scrollspy component callout):
+
+{{< ods-incompatibility-alert >}}
+This navbar version of the Scrollspy component is not compatible with the Orange Design System.
+
+Refer to [Orange navbar]({{< docsref "/components/orange-navbar" >}}) and [Nav & tabs]({{< docsref "/components/navs-tabs" >}}).
+{{< /ods-incompatibility-alert >}}
```

Comment:
Replace callout example by the one here: http://localhost:9001/docs/5.2/components/button-group/#checkbox-and-radio-button-groups.

This checkbox variant should not be used because it does not respect the Orange Design System specifications.

From the Orange Design System point of view, checkboxes should be represented as in our Boosted [Checks](http://localhost:9001/docs/5.2/forms/checks-radios/) component.

For the correct design, you can also refer to the [Checkbox](https://system.design.orange.com/0c1af118d/p/88ab5b-forms/b/599459/i/48901789) component on on the Orange Design System website.